.gitignore

```diff
@@ -113,3 +113,4 @@ $RECYCLE.BIN/
 
 .env
 storage/
+deploy_key
```

Comment:
New line ... motivation to start using VSCode... such stuff will be added automatically when u save the file hehe

Reply:
Oh yes. Atom adds new lines at the end of files too, and has Python linting abilities.